Microsoft.Toolkit.Uwp.UI/Extensions/TextBoxMask/TextBoxMask.Properties.cs

```diff
@@ -77,7 +78,7 @@ public static string GetPlaceHolder(DependencyObject obj)
         /// </summary>
         /// <param name="obj">TextBox Control</param>
         /// <param name="value">placeholder Value</param>
-        public static void SetPlaceHolder(DependencyObject obj, string value)
+        public static void SetPlaceHolder(TextBox obj, string value)
```

Comment:
Can you change the sample to ensure that we have use both x:Bind and Binding.. I have a feeling TextBox is going to bust x:Bind